packages/jest-worker/README.md

```diff
@@ -168,7 +168,7 @@ main();
 ### File `worker.js`
 
 ```javascript
-import babel from 'babel-core';
+import babel from '@babel/core';
```

Comment:
can you update the example in this doc to use https://babeljs.io/docs/en/babel-core#transformfileasync? Then we don't need the `new Promise` wrapper